### PR TITLE
Recommend using a rake task for FactoryGirl.lint

### LINKED ISSUE #772: When to run `FactoryGirl.lint`
The README says:

> Recommended usage of FactoryGirl.lint is to invoke this once before the test suite is run.

But doesn't that imply that even when running a single test, e.g. as part of TDD, every factory will be linted, causing a dramatic slowdown?

### PR DESCRIPTION
**Pull request by drapergeek**
Fixes #772

### CODE REVIEW

File: GETTING_STARTED.md [hunk 1]
```diff
@@ -828,22 +828,32 @@ FactoryGirl.lint
...
+will negatively impact the performance
+of your tests
+when running single tests.
+
+Example Rask task:
+
+```ruby
+# lib/tasks/factory_girl.rake
+namespace :factory_girl do
+  desc "Verify that all FactorGirl factories are valid"
```
Comment (andyw8) [inline GETTING_STARTED.md]: `FactoryGirl`

File: GETTING_STARTED.md [hunk 2]
```diff
@@ -828,22 +828,32 @@ FactoryGirl.lint
...
+  task lint: [:environment] do
```
Comment (andyw8) [inline GETTING_STARTED.md]: I think the square brackets are unnecessary here?

Comment (drapergeek) [inline GETTING_STARTED.md]: Indeed! I'll update it, thanks

**Comment by joshuaclayton** [comment]
@drapergeek thank you sir! I've merged this as f9cec9d326ff04a8d033515aa020c8a2ec4eb372

**Pull request closed without merging**